config/schedule.rb

```diff
@@ -7,3 +7,11 @@
 every :day, at: '3:00 am', roles: [:job_creator] do
   rake 'clear_expired_sessions'
 end
+
+every Figaro.env.queue_health_check_frequency_seconds.to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.check'
+end
+
+every Figaro.env.queue_health_check_frequency_seconds.to_i, roles: [:job_creator] do
+  runner 'WorkerHealthChecker.enqueue_dummy_jobs'
+end
```

Comment:
Is it possible to combine these in the same block? It looks like everything is the same except for the runner.